test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -2192,15 +2198,15 @@ class MultipleDisposalsTriageControllerSpec
       }
 
       "show an error page" when {
-        val taxYear = sample[TaxYear].copy(
-          startDateInclusive = LocalDate.of(2020, 4, 6),
-          endDateExclusive = LocalDate.of(2021, 4, 6)
+        val cutoffTaxYearDateRange = sample[TaxYear].copy(
```

Comment:
Is this date range correct? previous one seems to be cutoff year and cutoff_year + 1